tools.md

```diff
@@ -14,16 +14,52 @@ sidenav:
 ## USWDS Component Library in Figma
 
 ## Figma tokens plug-in
+#### What is Figma Tokens? ####
+The Figma Tokens plugin allows you to integrate tokens into your designs in Figma. There are three approaches to begin using the Tokens plugin:
+  1. Create your tokens from scratch, 
+  2. Import your existing styles into the Tokens plugin, 
+  3. Or create tokens using the built-in JSON editor. 
 
-- item
-- item
-- item
+#### Importing tokens ####
+- If you're working with an existing design system and you've already defined font styles, colors, spacing, etc. in Figma, then use the import feature to generate tokens based on those existing styles. 
+- This can be the easiest way to start using tokens because the plugin will auto-generate the tokens based on what you have in your system, instead of starting from nothing. 
 
-## Other Figma plug-ins
+![import button on tokens plugin](/assets/img/import-styles.jpeg)
+
+- After importing existing styles to create tokens, you can adjust the token names with your developer and come up with the naming conventions that make the most senese for you.
```

Comment:
Let's change this to say something about using the USWDS tokens. We understand they may not be starting with USWDS. But whatever they do moving forward, should use that language, not "conventions that make the most sense for you."